ui/page/transaction_details_page.go

```diff
@@ -184,6 +184,11 @@ func (pg *TransactionDetailsPage) txnBalanceAndStatus(gtx layout.Context) layout
 						amount := strings.Split(dcrutil.Amount(pg.transaction.Amount).String(), " ")
 						return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Baseline}.Layout(gtx,
 							layout.Rigid(func(gtx C) D {
+								if pg.transaction.Direction == dcrlibwallet.TxDirectionSent {
+									sentAmountLabel := pg.Theme.H4(amount[0])
+									sentAmountLabel.Text = "-" + amount[0]
+									return layout.Inset{Right: values.MarginPadding2}.Layout(gtx, sentAmountLabel.Layout)
+								}
 								return layout.Inset{Right: values.MarginPadding2}.Layout(gtx, pg.Theme.H4(amount[0]).Layout)
```

Comment:
Simplify
```suggestion
								amt := amount[0]
								if pg.transaction.Direction == dcrlibwallet.TxDirectionSent {
									amt = "-" + amount[0]
								}
								return layout.Inset{Right: values.MarginPadding2}.Layout(gtx, pg.Theme.H4(amt).Layout)
```

Reply:
This will throw an error because the initialised value of "amt" is not used 
I will simplify in another way